megamek/src/megamek/client/ui/swing/skinEditor/SkinSpecEditor.java

```diff
@@ -253,11 +253,16 @@ private void setupEditPanel() {
             udEditPanel.setupSkinEditPanel(SkinXMLHandler.getUnitDisplaySkin());
             editPanel.add(udEditPanel);
         } else {
-            SkinSpecification skinSpec = SkinXMLHandler.getSkin(selectedComp.getComp());
-            enableBorders.setSelected(!skinSpec.noBorder);
-            enableBorders.setEnabled(true);
-            skinEditPanel.setupSkinEditPanel(skinSpec);
-            editPanel.add(skinEditPanel);
+            // TODO : Tear out broken Skin Editor
+            try {
+                SkinSpecification skinSpec = SkinXMLHandler.getSkin(selectedComp.getComp());
+                enableBorders.setSelected(!skinSpec.noBorder);
+                enableBorders.setEnabled(true);
+                skinEditPanel.setupSkinEditPanel(skinSpec);
+                editPanel.add(skinEditPanel);
+            } catch (Exception ignored) {
+
```

Comment:
?

Reply:
If you mean the tear out note, it was a code misread.
Otherwise... there's a potential exception that can be thrown, but in all likelihood won't. I'll be doing a full look at catches in the future, and will probably change it then.